PowerValidatedSolutions.psm1

```diff
@@ -37451,6 +37568,2386 @@ Export-ModuleMember -Function Test-NtpServer
 #######################################################################################################################
 #Region                                D E P R E C A T E D  F U N C T I O N S                               ###########
 
+Function Join-ESXiActiveDirectory {
```

Comment:
Looks like the functions removed in #186 slipped back in.

Reply:
Fixed, good catch that's my bad